embrace-android-sdk/src/integrationTest/kotlin/io/embrace/android/embracesdk/session/SessionSpanTest.kt

```diff
@@ -47,4 +49,17 @@ internal class SessionSpanTest {
             })
         }
     }
+
+    @Test
+    fun `session span event do no affect logging maximum breadcrumbs`() {
```

Comment:
```suggestion
    fun `session span event limits do not affect logging maximum breadcrumbs`() {
```

Reply:
Crap. Didn't see this initially. I'll make the change